app/src/main/java/com/xzyht/notifyrelay/ui/pages/RemoteAppsPage.kt

```
@@ -0,0 +1,776 @@
+    val bitmap = remember(app.packageName) {
+        try {
+            val appInfo = packageManager.getApplicationInfo(app.packageName, 0)
+            val drawable = appInfo.loadIcon(packageManager)
+            val width = drawable.intrinsicWidth.takeIf { it > 0 } ?: 1
+            val height = drawable.intrinsicHeight.takeIf { it > 0 } ?: 1
+            val bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888)
+            val canvas = android.graphics.Canvas(bitmap)
+            drawable.setBounds(0, 0, width, height)
+            drawable.draw(canvas)
+            bitmap.asImageBitmap()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)